rasa/telemetry.py

```diff
@@ -563,6 +563,27 @@ def toggle_telemetry_reporting(is_enabled: bool) -> None:
 
     rasa_utils.write_global_config_value(CONFIG_FILE_TELEMETRY_KEY, configuration)
 
```

Comment:
In order to resolve both E302 errors, please leave 2 lines empty between two functions.